src/freenas/etc/directoryservice/LDAP/ctl

```diff
@@ -7,6 +7,7 @@ status_file="/var/run/directoryservice.ldap"
 service=/usr/sbin/service
 python=/usr/local/bin/python
 notifier=/usr/local/bin/midclt
+smbpasswd=/usr/local/bin/smbpasswd
```

Comment:
where is this used?

Reply:
I was originally going to set the ldap secret inside the ctl script, but then had a brief spurt of sanity. Will delete that line.